packages/calcite-components/src/components/block/block.tsx

```diff
@@ -207,11 +249,17 @@ export class Block
   //
   // --------------------------------------------------------------------------
 
-  /**
-   * Emits when the component's header is clicked.
-   */
-  @Event({ cancelable: false }) calciteBlockToggle: EventEmitter<void>;
```

Comment:
This would be a breaking change right? Do we want to have a breaking change here?

Reply:
Or should we leave it and deprecate it?